Show only pods from matching namespace in Grafana.

jgoclawski opened the pull request:
In "pods.json" dashboard there is a list of pods to select from.
This change filters the list, to contain only pods from previously selected namespace.

Fixes: #1396.

![zrzut ekranu z 2017-01-12 18-12-40](https://cloud.githubusercontent.com/assets/1043875/21900795/731c0118-d8f5-11e6-8809-f038249f6acc.png)

k8s-reviewable commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.kubernetes.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/heapster/1460)
<!-- Reviewable:end -->

jgoclawski committed:
Show only pods from matching namespace in Grafana.

In "pods.json" dashboard there is a list of pods to select from.
This change filters the list, to contain only pods from previously selected namespace.

Fixes: #1396.

luxas commented:
Thank you @jgoclawski, this is great!

LGTM (and extra plus for the gpg-signed commit! :smile:)

@piosz Please merge into the release

piosz commented:
@jgoclawski thanks for the contribution. @luxas thanks for the review.

jgoclawski commented:
Thanks for a quick merge!
